.github/workflows/tests.yml

```diff
@@ -0,0 +1,52 @@
+name: Tests
+
+on:
+  pull_request:
+    branches: [ main ]
+
+env:
+  LLVM_VERSION: 17
+  JLM_ROOT_DIR: /tmp/jlm
+  CIRCT_INSTALL_DIR: /tmp/circt
+
+jobs:
+
+  hls-test-suite:
+    runs-on: ubuntu-22.04
+    if: contains(github.event.pull_request.title, '[DisableCI]') == false
+    steps:
+    - uses: actions/checkout@v4
+    - name: "Install packages"
+      uses: ./.github/actions/InstallPackages
+      with:
+        llvm-version: 17
```

Comment:
I guess it is not possible to use the environment variable here, right? Is there a way to use something similar like `inputs.llvm-version` here like in the composite actions?

It seems there exist something similar that you can use: https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions#example-of-onworkflow_callinputs

Reply:
I didn't get the inputs to work. The documentation is for callable workflows, so maybe it only works when called from a different workflow.
I decided to move the MLIR symlinks to the InstallPackage action, which results in the LLVM version only having to be specified once.